bigquery/tests/unit/test_magics.py

```diff
@@ -65,6 +65,22 @@ def ipython_interactive(request, ipython):
         yield ipython
 
 
+@pytest.fixture(scope="session")
+def missing_bq_storage():
+    """Provide a patcher that can make the bigquery storage import to fail."""
+    orig_import = six.moves.builtins.__import__
+
+    def custom_import(name, globals=None, locals=None, fromlist=(), level=0):
```

Comment:
Should we make this `__import__` mock into a test utility (at https://github.com/googleapis/google-cloud-python/tree/master/test_utils/test_utils or maybe https://github.com/googleapis/google-cloud-python/blob/master/bigquery/tests/unit/helpers.py)?

Seems generally useful, and also something I'd like to see tests for.

Reply:
I'd say we put it in bigquery test utils for now, and ask other stakeholders if they are fine with promoting it to core test utils.
(there might be additional features/generalizations requested, but that would not block this particular PR)

Reply:
BTW, whom can I ask if the new helper is fit for promotion, who owns the core?

Reply:
> BTW, whom can I ask if the new helper is fit for promotion, who owns the core?
> 
> (if there isn't one and we can do it at our own discretion, please let me know)

I would say add a separate PR promoting it, and tag @busunkim96 for review.